src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,73 @@
+.dg_breadcrumbs {
+  color: $black;
+  display: inline-block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  list-style: none;
+  margin-bottom: 0px;
+  margin-top: $margin-small;
+  padding-bottom: $padding-smallest;
+  padding-left: 0px;
+  padding-top: 0px;
+  text-transform: uppercase;
```

Comment:
I think you can adjust .dg_breadcrumbs based on this

![image](https://user-images.githubusercontent.com/1933400/65598480-7dffff80-df69-11e9-9cac-685c37b64c32.png)

Reply:
As per feedback, removed styles listed and committed.